src/robusta/integrations/receiver.py

```diff
@@ -209,50 +220,53 @@ def __validate_request(self, action_request: ExternalActionRequest, validate_tim
             logging.error(
                 f"Rejecting incoming request because it's too old. Cannot verify request {action_request}"
             )
-            return False
+            return ValidationResponse(http_code=500, error_code=4500, error_msg="Illegal timestamp")
 
         signing_key = self.event_handler.get_global_config().get("signing_key")
         body = action_request.body
         if not signing_key:
             logging.error(f"Signing key not available. Cannot verify request {body}")
-            return False
+            return ValidationResponse(http_code=500, error_code=4501, error_msg="No signing key")
 
         # First auth protocol option, based on signature only
         signature = action_request.signature
         if signature:
             generated_signature = sign_action_request(body, signing_key)
-            return hmac.compare_digest(generated_signature, signature)
+            if hmac.compare_digest(generated_signature, signature):
+                return ValidationResponse()
+            else:
+                return ValidationResponse(http_code=500, error_code=4502,  error_msg="Signature missmatch")
```

Comment:
type in "missmatch", should be "mismatch"

Reply:
fixing